### ISSUE TITLE
changing the default stun server

### ISSUE THREAD

**Issue by amirtgm** [libp2p/js-libp2p-webrtc-star#455]
I used the configs when I create my libp2p node and added my custom turn server to it like this:

```js
transport: {
        WebRTCStar: {
          filter: filters.all,
          listenerOptions: {
            config: {
              iceServers: [
                {
                  urls: 'turn:*******',
                  username: '*****',
                  credential: '******',
                  credentialType: 'password',
                },
              ],
            },
          },
        },
      },
```
but after a long time of debugging, I found out that this customization is only happening for the listener part.
on the other hand, when a user tries to dial another one, there will be a new instance of simple peer with the default configuration.

I managed to resolve it by adding the configs to dial options but this shouldn't be the case right?
```js
private dial = async (addr: Multiaddr, protocol: Protocols) => {
    return this.node.dialProtocol(addr, protocol, {
      spOptions: {
        config: {
          iceServers: [
           {
                  urls: 'turn:*******',
                  username: '*****',
                  credential: '******',
                  credentialType: 'password',
                },
          ],
        },
      },
    } as any);
  };

```

**Comment by BaptisteHubert** [comment]
Hi @BigLep, since you previously triaged this issue, could you expand on the way it could be pushed forward?
Being in the "maintenance priorities", does that mean that you already pinned down the problem at the core of this issue?

**Comment by BigLep** [comment]
Hi @BaptisteHubert: Unfortunately I don't expect this issue to get any engagement from the core maintainers in light of the work that is happening with building a robust WebRTC implementation across libp2p implementations: https://github.com/libp2p/specs/pull/412

@p-shahi : I thought we had a work item for clarifying the maintenance posture for webrtc-star and webrtc-direct.  If not would you please be able to create one and execute on it?

**Comment by p-shahi** [comment]
Yes please see the incoming archival notice in the readme: https://github.com/libp2p/js-libp2p-webrtc-star/pull/486
I'll archive this repo shortly
